CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # ChangeLog
 
+## Unreleased
+
+- add `ft_sdk::email::sender_info_from_env` function to get email sender info
+  from environment variables.
```

Comment:
Not fond of this. I think we can update the "New Email Design”, to also includ a config.json in each `mid` folder, which will contain sender information (and subject, so we have one less file to manage). 

Basically I feel sender information is going to be “campaign” specific. 

Also we need reply-to, which may be different from sender. Which can also in in `<mid>/config.json`.